apps/nextjs/src/app/(main)/organizations/[org_slug]/page.tsx

```
@@ -37,21 +37,33 @@
 
   return (
     <>
-      <OrganizationHeader
-        organization={organization}
-        classNames={{ wrapper: "my-8" }}
+      <section
+        role="region"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)